resources/views/dashboard/training/tickets.blade.php

```diff
@@ -88,6 +88,8 @@
                                 <th scope="col">Session ID</th>
                                 <th scope="col">Start Time</th>
                                 <th scope="col">End Time</th>
+                                <th scope="col">Score(1-5)</th>
```

Comment:
For UX, we need to provide some context behind the score (ex. 5 is best, 1 is worst)

Reply:
Probably need a forced line break here so it displays a little nicer...
![image](https://github.com/user-attachments/assets/ce4e78f6-dede-4dcd-9cd6-340f06cc2232)

Reply:
Adding context seems to be too much for this tiny column... I was using (1 = Poor, 5 = Best). Any other suggestions?